stories/tab.stories.js

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react'
+import { storiesOf, addDecorator } from '@storybook/react'
+import Theme from './theme'
+import Tab, { Tabs } from '../src/components/Common/Tab'
+
+addDecorator(Theme)
+
+class MultipleTabs extends PureComponent {
```

Comment:
not exactly sure what you're suggesting.. is making the `Tabs` a stateful component?

Reply:
That's what I was suggesting. Until I saw that MultipleTabs was created by using Tabs inside it.

So I guess now I'm suggesting that we create this component (MultipleTabs) inside Common?

Reply:
Fair enough if you think it's outside the scope of this ticket, since this was just about making `Tab`. But we'll use this pattern once we create the About Us page & it might just be useful to have inside Common straight away?

Reply:
I feel like these 'common' components should be as stateless and dumb as possible. That way they're reusable more easily and the implementation is far more simpler...

Reply:
but yeah, if you want, create a card for it and we'll discuss its API!